test/config/integration/certs/cacert.pem

```diff
@@ -1,23 +1,23 @@
 -----BEGIN CERTIFICATE-----
-MIID0jCCArqgAwIBAgIJAJxgLCQiz7YlMA0GCSqGSIb3DQEBCwUAMHYxCzAJBgNV
```

Comment:
Can you clue me on on why these need to be updated?  I'm always mildly twitchy about changing testdata and code at the same time

Reply:
I needed to add "localhost" as a valid SAN for the upstream cert so I could test that path in the integration test. I then ran the script which regenerates all the certs. I can probably just revert all the cert changes other than the cert I changed. WDYT?

Reply:
SGTM